u3-views/app.js

```diff
@@ -2,7 +2,7 @@ const express = require('express')
 const expHbs = require('express-handlebars')
 const { courseDetails, courseUnits, courseStudents } = require('./features/courseController')
 const { index } = require('./features/indexController')
-const { Video, Videohome } = require('./features/videoController')
+const { Video, Videohome } = require('./features/videoController').lowerCamelcase()
```

Comment:
ฉันไม่คิดว่าคุณทดสอบรหัสนี้ คุณกำลังพยายามเปลี่ยนค่าเป็นตัวพิมพ์เล็กอูฐ แต่คุณค่าเป็นวัตถุ นี้จะพัง
I don't think you tested this code. You are attempting to change the _value_ to lower camel case; but the value is an object. This will crash.

ฉันขอให้คุณเปลี่ยนชื่อตัวแปรเป็นตัวพิมพ์เล็กของอูฐ:
I asked you to change the variable _name_ to lower camel case:
```
const { video, videoHome } = require('./features/videoController')
```